research/format.markdown

```diff
@@ -0,0 +1,138 @@
+## Draft 2 to Draft 3:
+
+> Behaviour: Not required to validate → MAY validate(but aren’t required to)
+
+**\-**  **street-address** : This should be a street address.
+
+**\-**  **locality** : This should be a city or town.
+
+**\-**  **region** : This should be a region (a state in the US, province in Canada, etc.)
+
+**\-**      **postal-code** : This should be a postal code (AKA zip code).
+
+**\-**      **country** : This should be the name of a country.
+
+
+**\+**     **host-name** : This SHOULD be a host-name.  
+
+
+
+## Draft 3 to Draft 4:
+
+> MAY validate → 
```

Comment:
Also above you started these blockquotes with `Behavior:` and here you didn't. Please be consistent throughout the document